org.eclipse.xtext.tests/src/org/eclipse/xtext/resource/XtextResourceTest.java

```diff
@@ -141,4 +141,12 @@ public void unloadRoot(EObject root) {
 		assertEquals(1, diag.getLength());
 	}
 
+	@Test public void testClearCache() throws Exception {
```

Comment:
is this a legit test ?!?

Reply:
If you make a teeny tiny change to the resource, e.g. resource.update(0, 0, " ") in line 147, then yes

Reply:
but with that (1) the tests seems green on master anyway and (2) the reconciler does a dummy edit only too. maybe you can have a look too

Reply:
any update @szarnekow ?